packages/nouns-sdk/src/contract/addresses.json

```diff
@@ -34,7 +34,7 @@
     "nounsAuctionHouseProxyAdmin": "0x5dAdD045E2aC09f55f49Bb84886Ea7724c675a0d",
     "nounsDaoExecutor": "0xc15008dE43D93D115BD64ED4D95817fFdBfb6DEA",
     "nounsDAOProxy": "0x22F7658f64be277e6b3968ecE7b773b092a39864",
-    "nounsDAOLogicV2": "0x3c793c2D7Cfd48046c75c63E43d36a2cB42fC649",
```

Comment:
@ripe0x why did you need to remove it?

Reply:
i think it got removed when i was testing the v2 contracts. added back in.